demos/bert_question_answering_embedding_demo/python/bert_question_answering_embedding_demo.py

```diff
@@ -120,55 +126,47 @@ def main():
         new_shapes = {}
         for i, input_info in ie_encoder_emb.input_info.items():
             new_shapes[i] = [1, length]
-            log.info("Reshaped input {} from {} to the {}".format(
+        try:
+            ie_encoder_emb.reshape(new_shapes)
+            log.debug("Reshape model {} from {} to the {}".format(
                 i,
                 input_info.input_data.shape,
                 new_shapes[i]))
-        log.info("Attempting to reshape the context embedding network to the modified inputs...")
-
-        try:
-            ie_encoder_emb.reshape(new_shapes)
-            log.info("Successful!")
         except RuntimeError:
             log.error("Failed to reshape the embedding network")
             raise
 
         # Loading model to the plugin
-        log.info("Loading model to the plugin")
         ie_encoder_exec_emb_dict[length] = ie.load_network(network=ie_encoder_emb, device_name=args.device)
+    log.info('Loaded model {} to {}'.format(model_xml_emb, args.device))
 
     # Read model for final exact qa
     if args.model_qa:
         model_xml = args.model_qa
         model_bin = model_xml.with_suffix(".bin")
-        log.info("Loading network files:\n\t{}\n\t{}".format(model_xml, model_bin))
+        log.info('Reading model {}'.format(model_xml))
 
         ie_encoder_qa = ie.read_network(model=model_xml, weights=model_bin)
         ie_encoder_qa.batch_size = 1
 
         input_names_qa = args.input_names_qa.split(',')
         output_names_qa = args.output_names_qa.split(',')
-        log.info("Expected input->output names: {}->{}".format(input_names_qa, output_names_qa))
+        log.debug("Expected input->output names: {}->{}".format(input_names_qa, output_names_qa))
 
         #check input and output names
         input_names_model_qa = list(ie_encoder_qa.input_info.keys())
         output_names_model_qa = list(ie_encoder_qa.outputs.keys())
-        log.info("Network input->output names: {}->{}".format(input_names_model_qa, output_names_model_qa))
+        log.debug("Network input->output names: {}->{}".format(input_names_model_qa, output_names_model_qa))
         if set(input_names_model_qa) != set(input_names_qa) or set(output_names_model_qa) != set(output_names_qa):
             log.error("Unexpected network input or output names")
             raise Exception("Unexpected network input or output names")
 
         # Loading model to the plugin
-        log.info("Loading model to the plugin")
         ie_encoder_qa_exec = ie.load_network(network=ie_encoder_qa, device_name=args.device)
+        log.info('Loaded model {} to {}'.format(model_xml, args.device))
 
         max_length_qc = ie_encoder_qa.input_info[input_names_qa[0]].input_data.shape[1]
 
-    #load vocabulary file for all models
-    log.info("Loading vocab file:\t{}".format(args.vocab))
-    vocab = load_vocab_file(args.vocab)
-    log.info("{} tokens loaded".format(len(vocab)))
-
```

Comment:
In previous [demo](https://github.com/openvinotoolkit/open_model_zoo/pull/2567/files#diff-e1b7287f0b22a67ffb9277271574d8c18475c94c28311fd92f4734269927ce9cR99) you log info about vocab.

Reply:
It is moved to the 92 line (https://github.com/openvinotoolkit/open_model_zoo/pull/2567/files#diff-67473656959ad9f97d22a3cb8d50119b84239180464f08f10f75171ef23a6cf0R92), before the Inference Engine initialization.